### PR TITLE
Loss function

### PR DESCRIPTION
**Pull request by capt-pyro**
Implemented the loss function with translation and rotation support. The reconstruction loss uses binary_cross_entropy_with_logits. Not BCEwithLogitsLoss.

### CODE REVIEW

**Comment by yannidd** [comment]
closes #7 #8

**Comment by yannidd** [comment]
forgot to merge :D
